saas/axops/src/ui/src/app/views/+timeline/commits-overview/commits-overview.component.ts

```diff
@@ -79,8 +80,9 @@ export class CommitsOverviewComponent implements OnChanges {
         }
     }
 
-    public selectCommit(revision: string): void {
-        this.expandedCommit = this.expandedCommit === revision ? null : revision;
+    public selectCommit(obj: {revision: string, repo: string}): void {
```

Comment:
Fields `expandedCommit` and `expandedCommitRepo` should be set to null only if both `obj.revision === this.expandedCommit` and `this.expandedCommitRepo === obj.repo`. Implemented change broke expand/collapse functionality. Try to select one commit and then select another without collapsing previous.

Reply:
Fixed